lib/node_modules/@stdlib/blas/ext/base/dsapxsumpw/lib/ndarray.js

```diff
@@ -75,57 +75,57 @@ function dsapxsumpw( N, alpha, x, stride, offset ) {
 	if ( N <= 0 ) {
 		return 0.0;
 	}
-	if ( N === 1 || stride === 0 ) {
-		return alpha + x[ offset ];
+	if ( N === 1 || strideX === 0 ) {
+		return alpha + x[ offsetX ];
```

Comment:
@headlessNode Don't we have the same `N * x[ offsetX ]` concerns here when `strideX = 0`? Meaning, shouldn't `x[ offsetX ]` be repeated `N` times?

Reply:
Yep!